Tasks/Gradle/task.loc.json

```diff
@@ -14,8 +14,8 @@
   ],
   "author": "Microsoft Corporation",
   "version": {
-    "Major": 1,
-    "Minor": 111,
+    "Major": 2,
```

Comment:
This is going to be preview task? but it's not marked as preview!
v2 version is being called out as preview task